### PR TITLE
fillets-ng, new recipe

### PR DESCRIPTION
**Pull request by Begasus**
checked for x86_gcc2, x86 and x86_64 ...
Thanks to @humdingerb  for the icon!

### CODE REVIEW

File: games-puzzle/fish-fillets/fillets_ng-1.0.1.recipe [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+SOURCE_URI_2="http://prdownloads.sourceforge.net/fillets/fillets-ng-data-$portVersion.tar.gz"
+CHECKSUM_SHA256_2="f0c979fb35ec550a43246fc209add8f45ca550a382c94d6383bb3f01b1073799"
+SOURCE_DIR="fillets-ng-1.0.1"
+ADDITIONAL_FILES="fillets.rdef.in"
+
+ARCHITECTURES="x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	fish_fillets$secondaryArchSuffix = $portVersion
```
Comment (korli) [inline games-puzzle/fish-fillets/fillets_ng-1.0.1.recipe]: This is different from the recipe filename.

Comment (korli) [inline games-puzzle/fish-fillets/fillets_ng-1.0.1.recipe]: Also the directory name and the recipe filename are different.

Comment (Begasus) [inline games-puzzle/fish-fillets/fillets_ng-1.0.1.recipe]: re-writing, keeping the gentoo portage site in mind ...

**Comment by Begasus** [comment]
Updated, hope this looks better ...

**Pull request merged**